res/TensorFlowPythonExamples/examples/unique/__init__.py

```diff
@@ -0,0 +1,4 @@
+import tensorflow as tf
+
+in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 6), name="unique_in")
+op_ = tf.compat.v1.math.unique(in_)
```

Comment:
this has error
```
AttributeError: module 'tensorflow._api.v1.compat.v1.math' has no attribute 'unique'
```

Reply:
Yes, the correct api is `tf.compat.v1.unique`, I fixed the line. Thanks for spotting.